src/file.js

```diff
@@ -5,10 +5,12 @@ export default (session, fileName) => {
   const file = {
     session,
     fileName,
+
     variables() {
       return this.session.client()
-        .then((client) => client.session.call('file.netcdf.variables', [fileName]));
+        .then((client) => client.session.call('cdat.file.variables', [fileName]));
```

Comment:
The `npm run build` command complained about this style of arrow function for me; I made it work by converting them to something like `(client) => { return client.session.call('cdat.file.variables', [fileName]); }`